### PR TITLE
[WIP] hardware wallet light applet installer

### PR DESCRIPTION
**Pull request by gravityblast**
[WIP description]

### CODE REVIEW

File: cmd/hardware-wallet-light/main.go [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+)
+
+type commandFunc func(*lightwallet.Installer) error
+
+var (
+	logger = log.New("package", "status-go/cmd/hardware-wallet-light")
+
+	commands map[string]commandFunc
+
+	flagCommand   = flag.String("c", "", "command")
```
Comment (dshulyak) [inline cmd/hardware-wallet-light/main.go:22]: it makes sense to use positional argument for commands

**Comment by divan** [comment]
Nice! Is there a reason why this code is not in a separate repository? On a first glance, there is nothing related to statusd code.

**Comment by gravityblast** [comment]
@divan yes this is actually a better idea. I was keeping it inside because it was easier to test it but now it's better to move it to a new repo.

**Comment by gravityblast** [comment]
closing this PR, all the code is now at https://github.com/status-im/hardware-wallet-go

<blockquote><div> GitHub</div><div><strong><a href="https://github.com/status-im/hardware-wallet-go">status-im/hardware-wallet-go</a></strong></div><div>Contribute to status-im/hardware-wallet-go development by creating an account on GitHub.</div></blockquote>

**Pull request closed without merging**